packages/utils/src/cmds/transform_translate.js

```diff
@@ -0,0 +1,381 @@
+        }
+
+        curSplitEscaped.match(/.{1,76}(?=(%5Cu))?/g).forEach(escapedSplit => {
+            parts.push(unescape(escapedSplit))
+        })
+
+        return parts
+    }, [])
+
+    return newParts
+}
+
+const getTemplateMainLanguage = () => `
+msgid ""
+msgstr ""
+"Project-Id-Version: i18next-conv\\n"
+"MIME-Version: 1.0\\n"
+"Content-Type: text/plain; charset=utf-8\\n"
+"Content-Transfer-Encoding: 8bit\\n"
+"Plural-Forms: nplurals=2; plural=(n != 1)\\n"
+"POT-Creation-Date: 2019-08-13T13:38:22.134Z\\n"
+"PO-Revision-Date: 2019-08-13T13:38:22.134Z\\n"
+`
+
+const getTemplateAlternativeLanguage = (appName, language) => `
+msgid ""
+msgstr ""
+"Project-Id-Version: PACKAGE VERSION\\n"
+"Report-Msgid-Bugs-To: \\n"
+"POT-Creation-Date: 2018-04-10 19:51+0000\\n"
+"PO-Revision-Date: YEAR-MO-DA HO:MI+ZONE\\n"
+"Last-Translator: FULL NAME <EMAIL@ADDRESS>\\n"
+"Language-Team: LANGUAGE <LL@li.org>\\n"
+"Language: ${language}\\n"
+"MIME-Version: 1.0\\n"
+"Content-Type: text/plain; charset=UTF-8\\n"
+"Content-Transfer-Encoding: 8bit\\n"
+"X-Generator: Translate Toolkit 2.2.5\\n"
+"X-Pootle-Path: /${language}/${appName}/${language}.po\\n"
+"X-Pootle-Revision: 29879\n"
+`
+
+exports.command = 'transform_translate'
+exports.describe = 'Transform old translation file style to new style'
+exports.builder = {
+    appName: {
+        describe:
+            'The name of the app, must be lower case, use dashes instead of whitespaces and be prefixed with "-app". Normally this should the url part of the github repo (maintenance app => "maintenance-app")',
+        type: 'string',
+    },
+
+    primaryLanguage: {
+        describe: 'Primary language',
+        type: 'string',
+        default: 'en',
+    },
+
+    languages: {
+        describe:
+            'A comma separated list of languages to create files for, when omitted, all are used',
+        type: 'string',
+    },
+
+    deleteOld: {
+        describe:
+            'Deletes the old translation files, use `--no-delete-old` to keep the old files',
+        type: 'boolean',
+        default: 'true',
+    },
+
+    outDir: {
+        describe: 'Output directory for new translation files',
+        type: 'string',
+        default: TRANSLATION_OUT_DIR,
+    },
+
+    inDir: {
+        describe: 'Input directory where old translation files can be found',
+        type: 'string',
+        default: TRANSLATION_IN_DIR,
+    },
+
+    createOutDir: {
+        describe:
+            'Create the output dir if not existing, use `--no-create-out-dir` to exit with 1 when folder does not exist',
+        type: 'boolean',
+        default: 'true',
+    },
+
+    overrideExistingFiles: {
+        describe: 'Overriding the contents of existing translation files',
+        type: 'boolean',
+        default: 'false',
+    },
+
+    appendToExistingFiles: {
+        describe:
+            'Appends the new contents to existing translation files, can only be used when not using `--override-existing-files`',
```

Comment:
Add `conflicts: 'overrideExistingFiles'` to let `yargs` enforce that this can't be used when `overrideExistingFiles` is also supplied.